plugins/inventory/hcloud.py

```diff
@@ -66,19 +66,26 @@
           default: ""
           type: str
           required: false
+        status:
+          description: Populate inventory with instances with this status.
+          default: []
+          type: list
```

Comment:
This should fix the sanity check :)
```suggestion
          type: list
          elements: str